terraform.tfvars.j2

```diff
@@ -0,0 +1,29 @@
+prefix = "{{ cluster_name }}"
+
+ansible_control_vm_flavor = "general.v1.small"
+ansible_control_vm_name   = "ansible-control"
+ansible_control_disk_size = 25
+
+seed_vm_flavor = "general.v1.small"
+seed_disk_size = 25
+
+multinode_flavor     = "general.v1.medium"
+multinode_image = "{{ multinode_image }}"
+multinode_keypair    = "MaxMNKP"
```

Comment:
This won't work for other people. Does Azimuth not provide a keypair name to use?

Reply:
This variable is one that should be provided by the user, as it should be a keypair for the user who the multinode is being created for. This is because the keypair which would be provided by Azimuth would create two issues: 

- The keypair name will likely belong to an already existing keypair, and this deployment hasn't been configured to deal with already existing keypairs. This will cause a Terraform/OpenStack error.
- The SSH key associated with the Azimuth's keypair will be one created by Azimuth prior to the multinode deployment. This is an SSH key which doesn't necessarily belong to the user and maybe shouldn't be the proprietary key for access to the instance. Azimuth only needs temporary and circumstantial access to the VM, potentially none at all if deploying just the infrastructure, whereas the user's access is a more continuous requirement.